src/ImageRunModal.jsx

```diff
@@ -1022,7 +1022,7 @@
                                   <Popover aria-label={_("Health check start period help")}
                                       enableFlip
                                       bodyContent={_("The initialization time needed for a container to bootstrap.")}>
-                                      <button onClick={e => e.preventDefault()} className="pf-c-form__group-label-help">
+                                      <button onClick={e => e.preventDefault()} className="pf-v5-c-form__group-label-help">
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-1299-20230606-080503-570e483c-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)